.jscsrc

```diff
@@ -0,0 +1,8 @@
+{
+    "preset": "airbnb",
+    "requirePaddingNewLinesAfterBlocks": false,
+    "requirePaddingNewLinesBeforeLineComments": false,
+    "safeContextKeyword": ["self"],
+    "requireTrailingComma": false,
+    "disallowEmptyBlocks": false
+}
```

Comment:
`Indent by 2 spaces.` and `A newline at the file end.` are already preset in `airbnb`.